modifiers/allocation/modifier.py

```diff
@@ -0,0 +1,204 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import math
+from enum import Enum
+from ramble.modkit import *
+
+
+class AllocOpt(Enum):
+    # Experiment resource requests
+    N_RANKS = 1
+    N_NODES = 2
+    N_CORES_PER_TASK = 3
+    N_THREADS = 4  # number of OMP threads per task
+    N_RANKS_PER_NODE = 5
+    N_GPUS = 6
+
+    # Descriptions of resources available on systems
+    GPUS_PER_NODE = 100
```

Comment:
we might want to default this to 2

Reply:
I think you mean in `configs/nosite-x86_64/variables.yaml` - yes I can do that

(although by default I was originally going to leave this blank we could that and through an error for any experiment that requests GPUs on `nosite-x86_64` - the systems that have GPUs we should set for sure)